### DISCUSSION TITLE
Push Notifications

### DISCUSSION THREAD

**Discussion by bitspent** [nhost/nhost#1138] [Q&A] [answered]
Does NHost Stack Support:

- push notifications ? pubsub? something similar to FCM.
- queuing based services (MQ or Kafka) 

**_Thank you!_**

**Comment by elitan** [comment, marked as answer]
Those two services you mention are not part of the Nhost stack. You can learn more about Nhost's architecture here: https://docs.nhost.io/#architecture

However, you can run any service alongside Nhost, but that you need to host yourself.
